test/js/service/github.service.spec.js

```diff
@@ -0,0 +1,32 @@
+describe("github.service", function() {
+
+  var mockWindow, mockNotification, sampleGithubService;
+  beforeEach(function(){
+    module(function($provide){
+      $provide.service('$window', function(){ 
+        this.location.href = jasmine.createSpy('location.href');
+      });
+      $provide.service('notifications', function(){
+        this.apiService.get = jasmine.createSpy('apiService.get');
+      });
+    });
+    module('services');
+  });
+
+  beforeEach(inject(function($window, notifications, githubService){
+    mockWindow=$window;
```

Comment:
we dont need this `$window` and `notifications`
what i would include here instead is `$httpBackend` service or simply `$injector`
and u can initialize it as such `$httpBackend = $injector.get('$httpBackend');`